src/bencher/args.rs

```diff
@@ -204,3 +204,83 @@ impl Display for KeyGeneratorType {
         )
     }
 }
+
+#[derive(Args, Clone)]
+pub(crate) struct BenchmarkCompactionArgs {
+    #[command(subcommand)]
+    pub(crate) subcommand: CompactionSubcommands,
+}
+
+#[derive(Subcommand, Clone)]
+#[command(about = "Benchmark SlateDB compaction.")]
+pub(crate) enum CompactionSubcommands {
+    Load(CompactionLoadArgs),
+    Run(CompactionRunArgs),
+    Clear(CompactionClearArgs),
+}
+
+#[derive(Args, Clone)]
+#[command(about = "Load test data.")]
+pub(crate) struct CompactionLoadArgs {
+    #[arg(
+        long,
+        help = "Size of each SSTable in bytes.",
+        default_value_t = 1_073_741_824
```

Comment:
We haven't really experimented with larger or smaller sizes. My main concern was structuring each level so that it's a range-partitioned set of SSTs rather than one SST so we can apply some config to drive down the size of indexes/filters (maybe just configuring that would be a better way to go, but it's more complicated to do).

There's a couple factors we have to keep in mind:
1. Larger SSTs have larger bloom filters and indexes. Generally you don't want these to get too big or they wind up displacing lots of stuff in cache. With a 1GB SST and 32 kb keys and 128 kb values you'd have ~8MB filters/indexes. I'm not sure if that's too big or not - we'll have to see if it causes problems when we do benchmarks and run real workloads.
2. More SSTs means that we produce a larger manifest, which gets expensive to read/write (which we do periodically to synchronize between the db and the compactor). SST info is the dominant factor in manifest size.

So what's there right now (the config and 1GB default) reflects that we want some way to limit manifest and sst metadata block sizes, and 1GB felt like a nice round number. We'll have to see if it becomes a problem and deal with  it then.